pylint/config/config_file_parser.py

```diff
@@ -31,8 +31,7 @@ def __init__(self, verbose: bool, linter: PyLinter) -> None:
         self.verbose_mode = verbose
         self.linter = linter
 
-    @staticmethod
-    def _parse_ini_file(file_path: Path) -> tuple[dict[str, str], list[str]]:
+    def _parse_ini_file(self, file_path: Path) -> tuple[dict[str, str], list[str]]:
```

Comment:
Tremor of the removal of ``no-self-use`` 😄 ?

Reply:
Yes!